website/source/docs/platform/k8s/helm.html.md

```diff
@@ -340,6 +349,35 @@ to run the sync program.
       The name of the private key for the certificate file within the
       `secretName` secret.
 
+  * <a name="v-connectinject-acl-bindingrule-selector" href="#v-connectinject-acl-bindingrule-selector">`namespaceSelector`</a> (`string: "serviceaccount.name!=default"`) -
+  A [selector](/docs/acl/acl-auth-methods.html#binding-rules) for restricting automatic injection to only matching services based on
+  their associated service account. By default, services using the `default` Kubernetes service account will not have a proxy injected.
+
+  * <a name="v-connectinject-centralconfig" href="#v-connectinject-centralconfig">`centralConfig`</a> - Values that configure
+  Consul's [central configuration](/docs/agent/config_entries.html) feature (requires Consul v1.5+ and consul-k8s v0.8.1+).
+
+      - <a name="v-connectinject-centralconfig-enabled" href="#v-connectinject-centralconfig-enabled">`secretName`</a> (`string: null`) -
```

Comment:
Should this really be `secretName`? That seems odd to me. If it's right, might want to do some explaining.

Reply:
Copy pasta error. Good catch!

Reply:
Fixed.